notebooks/official/model_monitoring/batch_prediction_model_monitoring.ipynb

```diff
@@ -0,0 +1,800 @@
+{
```

Comment:
<h2>Installation and Setup  -&gt; Installation</h2><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/723/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>fixed</p>